src/test/java/com/epam/tc/hw2/ex2/TestExercise2.java

```diff
@@ -0,0 +1,62 @@
+package com.epam.tc.hw2.ex2;
+
+import com.epam.tc.hw2.WebDriverSetup;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.testng.annotations.Test;
+
+public class TestExercise2 extends WebDriverSetup {
+    @Test
+    public void checkExercice2() {
+        SoftAssertions softly = new SoftAssertions();
+
+
+        //Open test site by URL
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html");
+
+        //Assert Browser title
+        softly.assertThat(driver.getTitle()).contains("Home Page");
```

Comment:
It's better to separate complicated tests with different types of verification steps to separate tests